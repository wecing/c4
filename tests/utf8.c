extern int printf(char *s, ...);

int main() {
  char *s = "你好";
  char *s2 = "\xEF\xBB\xBF";
  printf("%s means hello\n", s);
  printf("utf-8 header is ");
  while (*s2) {
    printf("\\x%X", *s2++ & 0xff);
  }
  printf("\n");
  return 0;
}
