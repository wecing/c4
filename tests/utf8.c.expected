你好 means hello
utf-8 header is \xEF\xBB\xBF
